packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1100,6 +1100,10 @@ export class ContainerRuntime
 				runtime.onSchemaChange(schema);
 			},
 		);
+		assert(
```

Comment:
@MarioJGMsoft we need to update this to a simple if check and then throw a `UsageError`.  We reserve asserts for only codepaths that we believe are impossible to hit no matter the inputs to the system.